examples/guidelines.md

```diff
@@ -0,0 +1,78 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Example Guidelines
+
+## An Example Is
+
+An example demonstrates running a real application/framework/workload
+on Kubernetes in a meaningful way. It is educational and informative.
+
+Examples are not:
+
+* Full app deployments, ready to use, with no explanation. These
```

Comment:
+1, in openshift we have examples for deploying things like DBs, and then we have examples for deploying a full application framework with a DB.  While you can argue the latter includes all the information of the former, it's much easier for someone looking to just deploy a new DB to start w/ the DB-only example.